### PR TITLE
Switch my Chrome OS media keys workaround from mouse keys to NKRO

### PR DESCRIPTION
**Pull request by bcat**
## Description

On further investigation, it seems that Chrome OS isn't processing media keys unless the USB endpoint with the consumer and system control devices also contains a keyboard or mouse device. Mouse keys aren't useful to me, and they actually seem to cause a Windows bug instead (#8323), so I'm switching to NKRO. This functions as an equivalent workaround for Chrome OS, and is also at least theoretically useful to me.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [X] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

None

## Checklist

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/crkbd/keymaps/bcat/keymap.c [hunk 1]
```diff
@@ -14,6 +14,7 @@ enum layer {
 
 #define KY_CSPC LCTL(KC_SPC)
 #define KY_LOCK LGUI(KC_L)
+#define KY_NKRO MAGIC_TOGGLE_NKRO
```
Comment (fauxpark) [inline keyboards/crkbd/keymaps/bcat/keymap.c:17]:
```suggestion
```

File: keyboards/crkbd/keymaps/bcat/keymap.c [hunk 2]
```diff
@@ -44,7 +45,7 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 
     /* Adjust layer: http://www.keyboard-layout-editor.com/#/gists/77e7572e077b36a23eb2086017e16fee */
     [LAYER_ADJUST] = LAYOUT(
-        _______,  _______,  KC_MPLY,  KC_VOLU,  KC_MSTP,  _______,                      EEP_RST,  RESET,    _______,  _______,  _______,  _______,
+        _______,  KY_NKRO,  KC_MPLY,  KC_VOLU,  KC_MSTP,  _______,                      EEP_RST,  RESET,    _______,  _______,  _______,  _______,
```
Comment (fauxpark) [inline keyboards/crkbd/keymaps/bcat/keymap.c:48]: No need for this custom define; you can use
```suggestion
        _______,  NK_TOGG,  KC_MPLY,  KC_VOLU,  KC_MSTP,  _______,                      EEP_RST,  RESET,    _______,  _______,  _______,  _______,
```
instead :)

Comment (bcat) [reply keyboards/crkbd/keymaps/bcat/keymap.c:48]: Cool, done here and in the other keymaps.

**Comment by zvecr** [comment]
Thanks!

**Pull request merged**